### PR TITLE
[documentation] How to add additional products to Automate

### PR DESCRIPTION
**Pull request by yzl**
### :nut_and_bolt: Description: What code changed, and why?
Describe how to add additional products to an existing Automate installation.

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [x] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: components/automate-chef-io/content/docs/infra-server.md [hunk 1]
```diff
@@ -156,6 +156,40 @@ Installations require elevated privileges, so run the commands as the superuser
...
+       [deployment.v1.svc]
+       products=["automate", "infra-server"]
+    ```
+
+2. Apply the patch to the Chef Automate installation:
+
+    ```shell
+       sudo chef-automate config ./patch.toml
+    ```
+   The command output shows the Chef Infra Server services being added:
```
Comment (msorens) [inline components/automate-chef-io/content/docs/infra-server.md:175]: Markdown lint reports that there should be a blank line on either side of a fenced code block or a list.
Same (pre-existing) issue on lines 231, 233, 237, 239, and 256 in this file, and several in the other file.

File: components/automate-chef-io/content/docs/infra-server.md [hunk 2]
```diff
@@ -156,6 +156,40 @@ Installations require elevated privileges, so run the commands as the superuser
 
 1. [Set up `knife`]({{< relref "infra-server.md#use-knife-with-chef-infra-server" >}}) for use with Chef Infra Server.
 
+## How to Add Chef Infra Server to a Chef Automate Installation
```
Comment (msorens) [inline components/automate-chef-io/content/docs/infra-server.md:159]: May want to review the usage of the words "How to" in the level-two titles here. I would think it should be used for all of them or for none of them.

**Comment by mjingle** [comment]
Waiting on feedback per https://chefio.slack.com/archives/C6UKR7SHZ/p1588980929436100 before reviewing.

**Comment by susanev** [comment]
@OBrienCommaJosh i believe this is waiting on your feedback here, when you have time

**Pull request merged**